src/languages/swift.js

```diff
@@ -491,14 +490,56 @@ export default function(hljs) {
     ]
   };
 
+  function noneOf(list) {
+    return negativeLookahead(either(...list));
+  }
+
+  const METHODS_ONLY = [...Swift.keywords, ...Swift.builtIns];
+  const FUNCTION_CALL = {
+    relevance: 0,
+    variants: [
+      {
+        // Functions and macro calls
+        scope: "title.function",
+        keywords: KEYWORDS,
```

Comment:
This seems weird, why would we have keywords popping up in the middle of a function name?  Or is this truly to cover "it's a function name that's also a keyword"?